README.md

```diff
@@ -1,10 +1,6 @@
-# OpenXLA Benchmark
+# Mobile Benchmark
 
-This is a home for the common benchmarking infrastructure described in the
-[accompanying RFC](https://github.com/openxla/community/blob/main/rfcs/20230510-common-benchmark-suite.md).
-It aims to be a common benchmark suite that is **compiler-agnostic** and can be
-used in standalone comparative benchmark workflows and regression benchmarking
-resident in each compiler project.
+This repository is focused on on-device ML benchmarks, using IREE codegen.
```

Comment:
Perhaps "This repository is focused on comparative benchmarks between IREE and other frameworks, both on-device and server."